docs/user-guide/global-configurations/tags-policy.md

```diff
@@ -1,51 +1,165 @@
 
-When mandatory tags are enabled, Devtron enforces the requirement to specify values for these tags during the application creation process. This ensures that all applications within the project adhere to the specified tag values.
+7. **Description** - Write a brief description explaining the significance of the tag.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/app-creation-with-tags.jpg)
+    ![Figure 8: Adding Description for the Tag](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/tag-description.gif)
 
-## Propagate Tags to Kubernetes Resources While Creating Application
+8. **Allow/Block Deployments** - (Not available for `Suggested tags`) Select the action to be taken if mandatory tags are not configured by the user in the intended projects:
+    * **Allow deployments** - Use this option if you want to allow the user to deploy an existing application where mandatory tags are not configured yet.
+    * **Block deployment stages of prod environments** - Use this option if you want to prevent the user from deploying an existing application to production environments, if mandatory tags are not configured. 
+    * **Block deployment stages of non-prod enviroments** - Use this option if you want to prevent the user from deploying an existing application to non-production environments, if mandatory tags are not configured. 
+    * **Block deployment stages of all environments** - This will prevent the user from deploying an existing application to all environments if mandatory tags are not configured.
 
-If tag propagation for a project is disabled globally, you can still enable it for individual applications. During the application creation process, you have the option to enable tag propagation specifically for that application. By doing so, the tags assigned to that application will be propagated as labels to the associated Kubernetes resources.
+    ![Figure 9: Deciding Deployment Restrictions](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/allow-block.gif)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/propagation-enable-disable.jpg)
+9. **Propagate Tag** - By default, tags assigned to applications in Devtron are not automatically propagated to Kubernetes resources as labels.
+
+    ![Figure 10a: Propagating Tags](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/propagate-tag.gif) 
+
+    <br />
+
+    ![Figure 10b: Enabling/Disabling Propagation](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/propagation-symbols.jpg) 
+    
+    <br />
+
```

Comment:
You do have defined how propagation works optionally we can attach labels doc from k8s

Reply:
Or any blog link of devtron if available